src/main/java/net/jonathangiles/tools/teenyhttpd/winter/Winter.java

```diff
@@ -0,0 +1,187 @@
+package net.jonathangiles.tools.teenyhttpd.winter;
+
+import net.jonathangiles.tools.teenyhttpd.TeenyHttpd;
+import net.jonathangiles.tools.teenyhttpd.winter.annot.*;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static net.jonathangiles.tools.teenyhttpd.model.Method.*;
+
+/**
+ * Winter teeny-framework to bootstrap a teeny server and his controllers
+ */
+public class Winter {
```

Comment:
I actually wonder if Winter can just be an implementation detail, and we can have API on TeenyHttpd:

```java
public class TeenyHttpd {
    public void addController(...);
    public void scanForControllers();
}
```

Reply:
Even cooler would be no API in TeenyHttpd, and just an annotation that users can put on their app that tells TeenyHttpd to scan for controllers.